beast/tools/run/make_ast_inputs.py

```diff
@@ -26,15 +27,17 @@
 importlib.reload(make_ast_xy_list)
 
 
-def make_ast_inputs(flux_bin_method=True):
+def make_ast_inputs(pick_method):
```

Comment:
In the docs, `pick_method` defaults to `flux_bin_method`, but it is not set to a default here. So, I'm not sure this will work.

Reply:
In the __name__ == "main", it passes "flux_bin_method" by default unless the user used either "random_seds" or "supplement_seds" argument. So, I think it should work.